### ISSUE TITLE
End-to-end baselines with currently supported models

### ISSUE THREAD

**Issue by wwwjn** [pytorch/torchtitan#2352]
-- From user feedback

Example:
Model: https://huggingface.co/Qwen/Qwen3-30B-A3B-Base
Data: https://huggingface.co/datasets/allenai/Dolci-Think-RL-7B-Completions-SFT 
Evals: https://huggingface.co/datasets/TIGER-Lab/MMLU-Pro 

The idea is that baseline run should cover data prep, training, checkpoint conversion to HF/vllm, and evaluation.
Ideally the experiment is hosted on a wandb, so that readers can track loss curves and eval results.

At the moment we have smth like this: https://github.com/pytorch/torchtitan/tree/main/torchtitan/experiments , which has no track of experimental result.

Note that these experiments have some readmes in it, but they do not have any performance numbers (not MFU etc., but quality of model predictions), with what one can compare with.

The idea is that such baseline will make it much easier to verify that training works as expected w.r.t. existing codebase we have if we can replicate titan’s baseline.
